.github/scripts/rust/Makefile.toml

```diff
@@ -18,6 +18,10 @@ CARGO_DOC_HACK_FLAGS = "--workspace"
 CARGO_RUSTDOC_FLAGS = "--all-features -Zunstable-options -Zrustdoc-scrape-examples=examples -- -Zunstable-options"
 CARGO_RUSTDOC_HACK_FLAGS = "--workspace"
 CARGO_TEST_FLAGS = ""
+# This needs to be here, as error-stack uses it, and _BASE **needs** to be defined before
+# the variable using it has been defined. This is done per suggestion of
+# https://github.com/sagiegurari/cargo-make/issues/685
+CARGO_TEST_HACK_FLAGS_BASE = ""
```

Comment:
This hotfix is no longer needed once [#688](https://github.com/sagiegurari/cargo-make/pull/688) lands. We could wait until then or just use the hotfix for now.

Reply:
I think we should keep the hot fix for now as this PR is preventing another workaround in #794, which is currently blocking us from releasing `error-stack@v0.2`